docs/source/parallel.rst

```diff
@@ -96,10 +96,13 @@ You can find more about custom joblib backends in the
 
 Note that :class:`joblib.Parallel` takes both a ``n_jobs`` and ``pre_dispatch``
 parameter. The Civis joblib backend doesn't queue submitted jobs itself,
-so it will run ``pre_dispatch`` jobs at once. The default value of
-``pre_dispatch`` is "2*n_jobs", which will run a maximum of ``2 * n_jobs`` jobs
-at once in the Civis Platform. Set ``pre_dispatch="n_jobs"`` in your
-:class:`~joblib.Parallel` call to run at most ``n_jobs`` jobs.
+so it will run ``pre_dispatch`` jobs at once.
+
+.. note::
+    The default value of
+    ``pre_dispatch`` is ``"2*n_jobs"``, which will run a maximum of ``2 * n_jobs`` jobs
+    at once on Civis Platform. Set ``pre_dispatch="n_jobs"`` in your
+    :class:`joblib.Parallel` call to run at most ``n_jobs`` jobs.
```

Comment:
I suggested using a "warning" when I filed #361, but on second thought a "note" would seem more appropriate actually.

Here's what the compiled docs would look like:

![Screen Shot 2021-12-06 at 9 49 43 AM](https://user-images.githubusercontent.com/86482098/144868475-a21f6f6e-5b31-4f86-b17d-86b7b19db82a.png)